CONTRIBUTING.md

```diff
@@ -1,7 +1,7 @@
 # Contributing to Hydra Lab
 
-The Hydra Lab team encourages community feedback and contributions. 
-Thank you for your interest in making Hydra Lab better!
+The Hydra Lab dev team would love community feedback and contributions. 
+Thank you for your interest in making mobile testing better with Hydra Lab!
 
```

Comment:
Here is also a `Contribute` part at `README.md`, do we need to update it as well?
![image](https://user-images.githubusercontent.com/30514480/209050490-2756ec01-4cb1-46f9-bd24-a3c862dbde60.png)

Reply:
Good point. I cleared the duplicate content and use link as a reference.